### PR TITLE
update vigs

### PR DESCRIPTION
**Pull request by kadyb**

### CODE REVIEW

File: vignettes/orthophotomap.Rmd [hunk 1]
```diff
@@ -46,7 +46,7 @@ Values close to 0 and below are related to water, bare soil surfaces or building
 
 
 ```r
-# attach packages
+# load packages
```
Comment (Nowosad) [inline vignettes/orthophotomap.Rmd:49]: It is not a correct term - it should be "attach".

File: vignettes/orthophotomap.Rmd.orig [hunk 2]
```diff
@@ -47,7 +47,7 @@ Values close to 0 and below are related to water, bare soil surfaces or building
 ## Analysis
 
 ```{r message=FALSE}
-# attach packages
+# load packages
```
Comment (Nowosad) [inline vignettes/orthophotomap.Rmd.orig:50]: Same here.

Comment (kadyb) [reply vignettes/orthophotomap.Rmd.orig:50]: Plus in **topodb.Rmd**. I will change it.

Comment (Nowosad) [reply vignettes/orthophotomap.Rmd.orig:50]: 👍🏻

Comment (kadyb) [reply vignettes/orthophotomap.Rmd.orig:50]: Or is it should be formally load and attach? From `library()` docs:

> library(package) and require(package) both load the namespace of the package with name package and attach it on the search list.

Comment (Nowosad) [reply vignettes/orthophotomap.Rmd.orig:50]: Good point. Probably both together are the most correct.

Comment (kadyb) [reply vignettes/orthophotomap.Rmd.orig:50]: Let's stay with your proposal to avoid confiusion.

**Comment by kadyb** [comment]
Should we do something with this warning (194 line)?

```r
bridges = st_geometry(st_intersection(rivers, roads))
#> Warning message:
#> attribute variables are assumed to be spatially constant throughout all geometries
```

**Comment by Nowosad** [comment]
We should not care about this warning.

**Pull request merged**